content/en/blog/2023/http-conventions-declared-stable.md

```diff
@@ -0,0 +1,264 @@
+---
+title: HTTP semantic conventions are declared stable
+linkTitle: HTTP semantic conventions
+date: 2023-11-03
+author: '[Trask Stalnaker](https://github.com/trask) (Microsoft)'
+cSpell:ignore: stalnaker trask
+---
+
+Earlier this year we embarked on a three month project to stabilize HTTP semantic conventions.
+
+We are proud to announce (nine months later), that the HTTP semantic conventions are now the first OpenTelemetry semantic convention to be declared stable!
+
+This stable version is a significant evolution over the previous unstable version.
+
+Because of the number of changes, and because of the number of users impacted by these changes, we have mapped out
+a transition plan for instrumentations to follow when updating to the stable version which prioritizes our users.
+
+We also want to share some of the reasons behind these changes, as we think that the stable version brings many
```

Comment:
I think that we need a section title just before here to give context to "the reasons behind these changes". Here's a suggestion (not the best, but it's all I can think of atm):

```suggestion
## Rational for reworked HTTP semconv

We also want to share some of the reasons behind these changes, as we think that the stable version brings many
```